new/LIT6557Mashafat.xml

```diff
@@ -49,7 +54,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
-            <div type="bibliography"/><!---->
+<div type="edition">
```

Comment:
I think that there are a few small problems here with the nesting of textparts. Do I understand it correctly, that the text is divided in the parts "Genealogyofkings", "GenealogyofMary", "GenealogyofEthiopiankings", and "LIT6568YanagastatKebr", and that you give the incipit for each of these textparts? If so, each of these textparts should be their own  `<div type="textpart">` and then contain a further  `<div type="textpart" subtype="incipit">` in which the incipit should be encoded. But maybe I misunderstood.